src/components/Register.jsx

```diff
@@ -32,6 +33,20 @@ console.assert(
 
 function Register({ credentials, setCredentials }) {
   const { setUserInfo } = useContext(UserContext);
+  const [passwordError, setPasswordError] = useState("");
```

Comment:
Can this be initialized with the validation status of the initial password?

Reply:
Better to do it later in the code after we call validatePassword function.